Tests/UnitTests/Core/Clusterization/TimedClusterizationTests.cpp

```diff
@@ -262,4 +300,72 @@ BOOST_AUTO_TEST_CASE(TimedGrid_2D_noTollerance) {
   }
 }
 
+BOOST_AUTO_TEST_CASE(TimedGrid_2D_duplicate_cells) {
+  // Cells with same position but time difference larger than tolerance should
+  // not be considered duplicates
+  CellCollection cells = {Cell(0ul, 0, 0, 0.0), Cell(1ul, 0, 0, 1.0)};
```

Comment:
I was under the impression that the readout will always only provide one value per cell and will have to choose which one to pass on.

If such a readout exists we should ultimately support it, but I worry a bit about implications for reconstruction without time information.